src/Storage/Commands.Storage/Blob/Cmdlet/SetAzureStorageBlobContent.cs

```diff
@@ -32,7 +32,7 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob
     /// <summary>
     /// download blob from azure
     /// </summary>
-    [Cmdlet(VerbsCommon.Set, StorageNouns.BlobContent, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = ManualParameterSet),
+    [Cmdlet(VerbsCommon.Set, StorageNouns.BlobContent, SupportsShouldProcess = true, DefaultParameterSetName = ManualParameterSet),
```

Comment:
What's the default ConfirmImpact?  We would like to have the ConfirmImpact= low for this cmdlet.